common.tf

```diff
@@ -104,9 +103,12 @@ module "dr2_developer_key" {
   source   = "git::https://github.com/nationalarchives/da-terraform-modules//kms"
   key_name = "${local.environment}-kms-dr2-dev"
   default_policy_variables = {
-    user_roles    = [data.aws_ssm_parameter.dev_admin_role.value, module.preservica_config_lambda.lambda_role_arn]
+    user_roles = [
+      data.aws_ssm_parameter.dev_admin_role.value,
+      module.preservica_config_lambda.lambda_role_arn,
```

Comment:
There is an extra comma here, is that deliberate?

Reply:
No, it's valid but trailing commas are annoying. I've removed it.